scripts/qa_crowbarsetup.sh

```diff
@@ -2187,6 +2191,13 @@ function oncontroller_testsetup()
         fi
     fi
 
+    if [[ -n "$wantdocker" ]] ; then
```

Comment:
This is making a huge function even longer.  It would be better if it called a separate function.